assets/js/blocks/cart/inner-blocks/proceed-to-checkout-block/edit.tsx

```diff
@@ -60,12 +68,24 @@ export const Edit = ( {
 					/>
 				) }
 			</InspectorControls>
-			<Noninteractive>
-				<Block
-					checkoutPageId={ checkoutPageId }
-					className={ className }
+			<Button className="wc-block-cart__submit-button">
+				<RichText
+					multiline={ false }
+					allowedFormats={ [] }
+					value={ buttonLabel }
+					onChange={ ( content ) => {
+						if ( content === '' ) {
+							setAttributes( {
+								buttonLabel: defaultButtonLabel,
+							} );
+						} else {
+							setAttributes( {
+								buttonLabel: content,
+							} );
+						}
+					} }
```

Comment:
I don't understand this effort, why are we setting the attribute back to its default value? I understand that you're trying to protect against people having zero value but couldn't this be achieved using the built in attribute handling in Gutenberg?

Reply:
That's what I tried initially, but if you have an empty string, and save the attribute, it will save it with the empty string, not the default value. So this counters that and doesn't let you save the attribute if it's empty.